static/app/components/events/interfaces/spans/index.tsx

```diff
@@ -69,23 +63,13 @@ function TraceErrorAlerts({
       <Alert type={getCumulativeAlertLevelFromErrors(errors)}>
         <ErrorLabel>{label}</ErrorLabel>
 
-        <AnchorLinkManager.Consumer>
-          {({scrollToHash}) => (
-            <TraceErrorList
-              trace={parsedTrace}
-              errors={errors}
-              onClickSpan={(event, spanId) => {
-                return scrollToSpan(spanId, scrollToHash, location, organization)(event);
-              }}
-            />
-          )}
-        </AnchorLinkManager.Consumer>
+        <TraceErrorList trace={parsedTrace} errors={errors} onClickSpan={() => {}} />
```

Comment:
`onClickSpan` here used to allow you to click on the error link that appears in the header of event details if there is an error in the event, clicking on it would scroll down to the span.

I've done some restructuring in this PR that changed AnchorLinkManager which would no longer make this possible, as a side effect of how anchoring works on a virtualized list. Fixing this is non trivial, and I don't want to contribute to further PR bloat so I can come back to fixing this a bit later 😛 

![image](https://user-images.githubusercontent.com/16740047/202795434-daa9fb40-384b-4086-b87d-68d1de1e0d22.png)